.ci/CI/src/integTestGen.jl

```diff
@@ -225,73 +237,73 @@ function get_package_info()::Dict{String,PackageInfo}
+#     else
+#         target_branch = get_target()
+#     end
+
+#     package_infos = get_package_info()
+
+#     # custom commit message variable can be set as first argument
+#     if length(ARGS) < 1
+#         extract_env_vars_from_git_message!(package_infos)
+#     else
+#         extract_env_vars_from_git_message!(package_infos, ARGS[1])
+#     end
+
+#     modify_package_url!(package_infos)
+#     modified_pkg = modified_package_name(package_infos)
+
+#     # TODO(SimeonEhrig): refactor me, that the conversion is not required anymore
+#     custom_urls = Dict{String,String}()
+#     for (name, info) in package_infos
+#         if info.modified_url != ""
+#             custom_urls[name] = info.modified_url
+#         end
+#     end
+#     qed_path = mktempdir(; cleanup=false)
+#     compat_changes = Dict{String,String}()
+
+#     pkg_tree = build_qed_dependency_graph!(qed_path, compat_changes, custom_urls)
+#     depending_pkg = IntegrationTests.depending_projects(
+#         modified_pkg, collect(keys(package_infos)), pkg_tree
+#     )
+
+#     job_yaml = Dict()
+
+#     if !isempty(depending_pkg)
+#         for p in depending_pkg
+#             # Handles the case of merging in the main branch. If we want to merge in the main branch, 
+#             # we do it because we want to publish the package. Therefore, we need to be sure that there 
+#             # is an existing version of the dependent QED packages that works with the new version of 
+#             # the package we want to release. The integration tests are tested against the development 
+#             # branch and the release version.
+#             #  - The dev branch version must pass, as this means that the latest version of the other 
+#             #    QED packages is compatible with our release version.
+#             #  - The release version integration tests may or may not pass. 
+#             #    1. If all of these pass, we will not need to increase the minor version of this package. 
+#             #    2. If they do not all pass, the minor version must be increased and the failing packages 
+#             #    must also be released later with an updated compat entry.
+#             #    In either case the release can proceed, as the released packages will continue to work
+#             #    because of their current compat entries.
+#             if target_branch == "main" && TargetBranch.is_pull_request()
+#                 generate_job_yaml!(p, "dev", ENV["CI_PROJECT_DIR"], job_yaml, package_infos)
+#                 generate_job_yaml!(
+#                     p, "main", ENV["CI_PROJECT_DIR"], job_yaml, package_infos, true
+#                 )
+#             else
+#                 generate_job_yaml!(
+#                     p, target_branch, ENV["CI_PROJECT_DIR"], job_yaml, package_infos
+#                 )
+#             end
+#         end
+#     else
+#         generate_dummy_job_yaml!(job_yaml)
+#     end
+#     println(YAML.write(job_yaml))
+# end
```

Comment:
Can this be deleted?

Reply:
Yes. The comment is a just a reminder what the original script was doing. I will remove it when everything is working.